### DISCUSSION TITLE
ancora problemi meross_lan

### DISCUSSION THREAD

**Discussion by barbatrukko** [indomus/forum#1582] [HUB: Home Assistant]
Salve, io continuo ad avere problemi con un paio di device meross.
La nuova versione del componente doveva aver risolto invece ancora nulla:
Accedo ad HA, mi vengono presentati i due nuovi device, clikko per aggiungerli, gli facio prendere la key dal cloud, ma dopo ne con hack ne con user riesco a fargliela digerire. Ho anche editato il file core.config_entries per eliminar evecchie istanze di quei due dispositivi... ma nulla.
Ci sono soluzioni? Alternative?

**Comment by Scialla** [comment]
l'alternativa è il custom component di Geniola, io lo uso, da un sacco di errori all'avvio ma, per ora, funziona (ma ho solo una ciabatta Meross)

**Reply by barbatrukko** [reply]
Ha problemi con le lampade RGB. Quando vai ad esempio nelle scene, selezioni la lampada e ti si apre la wizard: passi da RGB a temperatura colore, setti la luminosità, salvi ma lui rimane in RGB alla configurazione precedente.
